tests/unit/test_gff3_parser.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
I initially put this test in `tests/datadriven`, but moved it once I realized that all tests there assume that the entity being tested is a ga4gh datamodel member - and all the code mechanisms for those tests fundamentally rely on that assumption. In contrast, this file really does represent just a unit test for the GFF3 parser, whose internal representation of feature data is not quite in alignment with GA4GH datamodels. Thus the inclusion in `unit`, not `datadriven`.